packages/coordinator/src/components/Popup/Popup.stories.tsx

```diff
@@ -0,0 +1,63 @@
+import { Story } from "@storybook/react";
+import { useState } from "react";
+import Button from "../Button";
+import { Popup, PopupProps } from "./Popup";
+
+export default {
+  component: Popup,
+  title: "Components/Popup",
+};
+
+const props = {
+  content: "האם ברצונך להסיר תורם זה  מהתור הנוכחי?",
+  primaryButtonText: "הסר תורם",
+  cancelButtonText: "השאר תורם",
+};
+
+const Template: Story<PopupProps> = (args) => {
+  const [open, setOpen] = useState(false);
+  const close = async () => setOpen(false);
+
+  return (
+    <div style={{ margin: 20 }}>
```

Comment:
Something I discovered recently - 
Instead of adding padding like this, you can add to the exported default in line 8:
`parameters: { layout: "padded" }`